### PR TITLE
MAUI-848170-Update User Guide APIs for Vol 3 API Reference Links_New

### PR DESCRIPTION
**Pull request by SuryaKaran2143**
**Feature description**

Update the API reference for the APIs released in Volume 3 main release

### CODE REVIEW

File: MAUI/Cartesian-Charts/Get-the-touch-position-in-chart.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ documentation: ug
 
 ChartInteractiveBehavior provides the following override methods to get the x and y positions when touching the [`SfCartesianChart`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Get-the-touch-position-in-chart.md:12]: Add API reference link for ChartInteractiveBehavior

Comment (SuryaKaran2143) [reply MAUI/Cartesian-Charts/Get-the-touch-position-in-chart.md]: I attached the API for the ChartInterActiveBehaviour class

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 2]
```diff
@@ -292,13 +292,13 @@ chart.ZoomPanBehavior = zooming;
...
+* [SelectionRectStrokeDashArray](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartZoomPanBehavior.html#Syncfusion_Maui_Charts_ChartZoomPanBehavior_SelectionRectStrokeDashArray) - Get or set the stroke dashes for selection rectangle.
 
-* [SelectionRectFill]() - Get or set the fill color for the selection rectangle.
+* [SelectionRectFill](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartZoomPanBehavior.html#Syncfusion_Maui_Charts_ChartZoomPanBehavior_SelectionRectFill) - Get or set the fill color for the selection rectangle.
 
 ### Show trackball axis label
 The selection zooming trackball axis label is enabled by setting the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) property to `true`. The default value of the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) is `false`. The TrackballLabelStyle[] property provides to customize the trackball axis labels. These options are:
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:304]: Update API reference links for trackball axis label properties

File: MAUI/Cartesian-Charts/Zooming-and-panning.md [hunk 3]
```diff
@@ -292,16 +292,16 @@ chart.ZoomPanBehavior = zooming;
...
+The selection zooming trackball axis label is enabled by setting the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) property to `true`. The default value of the [ShowTrackballLabel](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_ShowTrackballLabel) is `false`. The [TrackballLabelStyle](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_TrackballLabelStyle) property provides to customize the trackball axis labels. These options are:
 
 * `Background`, of type `Brush`, describes the background color of the labels.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Cartesian-Charts/Zooming-and-panning.md:306]: Add API reference for TrackballLabelStyle individual properties

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine.

**Pull request merged**